src/java/htsjdk/variant/variantcontext/VariantContext.java

```diff
@@ -721,33 +721,125 @@ public String getID() {
+     */
     public Set<String> getFilters()             { return commonInfo.getFilters(); }
+    
+    /** shortcut to {@link CommonInfo#isFiltered()}
+     * @return true if any filter been defined  <code>(!filters.isEmpty())
+     */
     public boolean isFiltered()                 { return commonInfo.isFiltered(); }
+    
+    /** shortcut to {@link CommonInfo#isNotFiltered()} */
     public boolean isNotFiltered()              { return commonInfo.isNotFiltered(); }
+    
+    /** shortcut to {@link CommonInfo#filtersWereApplied()}
+     *  @return true if no filter has been defined  <code>(getFiltersMaybeNull()==null)</code>
+     */
     public boolean filtersWereApplied()         { return commonInfo.filtersWereApplied(); }
+    
+    /** shortcut to {@link CommonInfo#hasLog10PError()}
+     * @return true if log10-based error estimate has been set 
+     */
     public boolean hasLog10PError()             { return commonInfo.hasLog10PError(); }
+    
+    /** shortcut to {@link CommonInfo#getLog10PError()}
+     * @return the -1 * log10-based error estimate
+     */
     public double getLog10PError()              { return commonInfo.getLog10PError(); }
+    
+    /** shortcut to {@link CommonInfo#getPhredScaledQual}
+     *  @return double - Phred scaled quality score
+     */
     public double getPhredScaledQual()          { return commonInfo.getPhredScaledQual(); }
-
+    
+    /** shortcut to {@link CommonInfo#getAttributes()}
+     * @return an unmodifiable Map of attributes
+     */
     public Map<String, Object>  getAttributes() { return commonInfo.getAttributes(); }
+    
+    /** shortcut to {@link CommonInfo#hasAttribute(String)}
+     * @return true if the key is present
+     */
     public boolean hasAttribute(String key)     { return commonInfo.hasAttribute(key); }
+    
+    /** shortcut to {@link CommonInfo#getAttribute(String)}
+     * @return the attribute value for the given key (or null if not set)
+     */
     public Object getAttribute(String key)      { return commonInfo.getAttribute(key); }
-
+    
+    /** shortcut to {@link CommonInfo#getAttribute(String, Object)}
+     * @return the attribute value for the given key (or defaultValue if not set)
+     */
     public Object getAttribute(String key, Object defaultValue) {
         return commonInfo.getAttribute(key, defaultValue);
     }
-
+    
+    /** shortcut to {@link CommonInfo#getAttributeAsString(String, String)}
+     * return an attribute as a String.
+     * if given key is not found the defaultValue is returned.
+     * if the value is not a String <code> String.valueOf(x)</code> is returned.
+     * @return the attribute as a String
+     */
     public String getAttributeAsString(String key, String defaultValue)   { return commonInfo.getAttributeAsString(key, defaultValue); }
+    
+    /** shortcut to {@link CommonInfo#getAttributeAsInt(String, int)}
+     * if given key is not found the defaultValue is returned.
```

Comment:
All the comments I made in `CommonInfo` for these `getAttributeAs*` methods should be applied here as welll.